docs/source/en/using-diffusers/loading_adapters.md

```diff
@@ -506,22 +506,11 @@ import torch
 from diffusers import StableDiffusionPipeline, DDIMScheduler
 from diffusers.utils import load_image
 
-noise_scheduler = DDIMScheduler(
-    num_train_timesteps=1000,
-    beta_start=0.00085,
-    beta_end=0.012,
-    beta_schedule="scaled_linear",
-    clip_sample=False,
-    set_alpha_to_one=False,
-    steps_offset=1
```

Comment:
Why?

Reply:
so the user doesn't have to remember the config, they can just use `from_config` instead - it uses a DDIM but with the same config as the default scheduler